nbgitpuller/handlers.py

```diff
@@ -194,7 +198,9 @@ def get(self):
             'repo': repo_url,
             'branch': self.get_argument('branch', 'gh-pages'),
             'depth': self.get_argument('depth'),
-            'subPath': self.get_argument('path')
+            'repodir': self.get_argument('repodir'),
+            'subPath': self.get_argument('path'),
+            'targetPath': self.get_argument('targetpath')
```

Comment:
I'm not grasping the full picture, but I notice that there is no check for targetpath / targetPath here, while there were above.

Reply:
I'm not sure either, but since it's a new parameter it seems reasonable to only support it for the current handlers so I've removed it from `LegacyInteractRedirectHandler`